### PR TITLE
[msbuild] Zip test results and log files for surfacing in jenkins

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

File: scripts/ci/run-test-mac-sdk.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ export PATH=${MONO_REPO_ROOT}/external/bockbuild/stage/bin:$PATH
 # Bundled MSBuild
 cd ${MONO_REPO_ROOT}/external/bockbuild/builds/msbuild-15/
 ${TESTCMD} --label="msbuild-tests" --timeout=180m ./build.sh -host mono -configuration Release
+zip msbuild-test-results.zip artifacts/2/Release-MONO/TestResults/*
```
Comment (akoeplinger) [inline scripts/ci/run-test-mac-sdk.sh]: this should use `${MONO_REPO_ROOT}/msbuild-test-results.zip` so it ends up in the repo root

Comment (radical) [reply scripts/ci/run-test-mac-sdk.sh]: Ok. How do I get the same path in `msbuild.py`, to copy the binlogs?

**Comment by akoeplinger** [comment]
@monojenkins build OS X Extended

File: scripts/ci/run-test-mac-sdk.sh (hunk 1 above)
Comment (akoeplinger) [reply scripts/ci/run-test-mac-sdk.sh]: I actually don't know, we probably need to bug Alexis about it :)

File: packaging/MacSDK/msbuild.py [hunk 2]
```diff
@@ -7,6 +7,7 @@ def __init__ (self):
 
 	def build (self):
 		self.sh ('./build.sh -host mono -configuration Release -skipTests')
+		self.sh ('zip msbuild-bin-logs.zip artifacts/Release-MONO/log/*')
```
Comment (radical) [inline packaging/MacSDK/msbuild.py]: @akoeplinger @alexischr  How do I get the path for `mono_repo` here, short a hardcoded relative path?

**Pull request merged**